.github/workflows/dry-run.yml

```
@@ -0,0 +1,87 @@
+name: sync-team dry-run
+
+on:
+  workflow_run:
+    workflows: [ CI ]
+    types:
+      - completed
+
+jobs:
+  dry-run:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' && github.event.workflow_run.event == 'pull_request' }}
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          repository: rust-lang/sync-team
```

```suggestion
          repository: rust-lang/sync-team
          persist-credentials: false
```
If we don't use the git token we can set this for additional security